tx_test.go

```diff
@@ -256,16 +452,17 @@ func TestTx_Change(t *testing.T) {
 		assert.NoError(t, err)
 
 		assert.Equal(t,
-			"0100000001760595866e99c1ce920197844740f5598b34763878696371d41b3a7c0a65b0b70000000000ffffffff03f4010000000000001976a9147a1980655efbfec416b2b0c663a7b3ac0b6a25d288ac000000000000000011006a02686903686f770361726503796f757a010000000000001976a91484e50b300b009833b297dc671817c79b5459da1d88ac00000000",
+			"0100000001760595866e99c1ce920197844740f5598b34763878696371d41b3a7c0a65b0b7000000006a473044022054ec562aefb1d5b4483906b0fa728a951b81944d8a82c2be1c6e7ea67e8c34b702200c2657c5d2a744d138bfb4204c579de20e984d680695823cba3fb7a0346b3c19412102c8803fdd437d902f08e3c2344cb33065c99d7c99982018ff9f7219c3dd352ff0ffffffff03f4010000000000001976a9147a1980655efbfec416b2b0c663a7b3ac0b6a25d288ac000000000000000011006a02686903686f770361726503796f757a010000000000001976a91484e50b300b009833b297dc671817c79b5459da1d88ac00000000",
 			tx.ToString(),
 		)
 
-		// todo: this tx produces the WRONG fee/change
-		/*
-			Size 145 B
-			Fee Paid 0.00000122 BSV
-			(0.00001 BSV - 0.00000878 BSV)
-			Fee Rate 0.8414 sat/B
-		*/
+		feePaid := tx.GetTotalInputSatoshis() - tx.GetTotalOutputSatoshis()
+		assert.Equal(t, uint64(122), feePaid)
+
+		txSize := len(tx.ToBytes())
+		assert.Equal(t, 251, txSize)
+
+		feeRate := float64(feePaid) / float64(txSize)
+		assert.Equal(t, 0.4860557768924303, feeRate)
```

Comment:
Am I wrong, this should be 0.50 or greater? A miner would not accept this tx?

Reply:
@jadwahab

Reply:
it's got some data in there (which is priced at 0.25) which is why it's slightly less than 0.5